src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -244,12 +266,13 @@ internal enum NodeFlags : byte
             ContainsSkippedText = 1 << 3,
             ContainsAnnotations = 1 << 4,
             IsNotMissing = 1 << 5,
+            ContainsAttributes = 1 << 6,
```

Comment:
I am assuming some mechanism already exists which is propagating this flag from the attribute up through its parents in the syntax tree? And that existing tests of ForAttributeWithMetadataName are helping us verify that this flag is present in the places we expect?